tests/python/contrib/test_hexagon/test_models.py

```diff
@@ -19,6 +19,7 @@
 import sys
 import pytest
 import numpy as np
+import onnx
```

Comment:
What version of onnx is needed?

Reply:
We install it from this file:
https://github.com/apache/tvm/blob/main/docker/install/ubuntu_install_onnx.sh
So that's 1.9